includes/admin/tools/export/class-give-export-donations.php

```diff
@@ -283,7 +283,8 @@ public function html() {
 						<td>
 							<?php wp_nonce_field( 'give_ajax_export', 'give_ajax_export' ); ?>
 							<input type="hidden" name="give-export-class" value="Give_Export_Donations_CSV"/>
-							<input type="submit" value="<?php _e( 'Generate CSV', 'give' ) ?>" class="give-export-donation-button button button-primary">
+							<input type="button" value="<?php esc_attr_e( 'Deselect All Fields', 'give' ); ?>" data-value="<?php esc_attr_e( 'Select All Fields' ); ?>" class="give-toggle-checkbox-selection button button-secondary">
```

Comment:
@Sidsector9 why did you forget to pass language domain in `<?php esc_attr_e( 'Select All Fields' ); ?>`

Reply:
@Sidsector9 I also found `39 ERRORS AND 19 WARNINGS` on code sniffing.